test/test.js

```diff
@@ -1058,14 +1058,50 @@ test('change process.cwd() to a test\'s directory', function (t) {
 
 test('Babel require hook only applies to the test file', function (t) {
 	execCli('fixture/babel-hook.js', function (err, stdout, stderr) {
-		t.ok(/exited with a non-zero exit code/.test(stderr));
-		t.ok(/Unexpected token/.test(stdout));
+		t.ok(/Unexpected token/.test(stderr));
 		t.ok(err);
 		t.is(err.code, 1);
 		t.end();
 	});
 });
 
+test('Unhandled promises will be reported to console', function (t) {
+	execCli('fixture/loud-rejection.js', function (err, stdout, stderr) {
+		t.ok(err);
+		t.ok(/You can't handle this/.test(stderr));
+		t.ok(/1 unhandled rejection[^s]/.test(stderr));
+		t.end();
+	});
+});
+
+test('uncaught exception will be reported to console', function (t) {
+	execCli('fixture/uncaught-exception.js', function (err, stdout, stderr) {
+		t.ok(err);
+		t.ok(/Can't catch me!/.test(stderr));
+		// TODO: This should get printed, but we reject the promise (ending all tests) instead of just ending that one test and reporting.
```

Comment:
cool, I'm assuming you took care of it in the merge? Or do you want me to go handle it now?